README.md

```diff
@@ -118,17 +124,62 @@ error = p.show(p.add(p.len(p.int(5)), p.int(3)))
 print(error)   # Output: len(5)+3 contains ill-typed subterm len(5)
 ```
 
+### JavaScript/TypeScript
+
+```javascript
+import pyfinalo from 'pyfinalo_js';
+
+// Direct evaluation
+const result = pyfinalo.show(pyfinalo.add(pyfinalo.int(5), pyfinalo.int(3)));
+console.log(result); // Output: "8 : int"
+
+// String operations
+const strResult = pyfinalo.show(pyfinalo.len(pyfinalo.str("hello")));
+console.log(strResult); // Output: "5 : int"
+
+// Complex expressions
+const complex = pyfinalo.show(
+  pyfinalo.add(
+    pyfinalo.mul(pyfinalo.int(2), pyfinalo.int(3)),
+    pyfinalo.int(4)
+  )
+);
+console.log(complex); // Output: "10 : int"
+```
+
 ## Available Interpreters
 
+### Python Modules
 - **`pyfinalo`**: Direct value interpreter (`DirectValInterp`) - evaluates expressions immediately
 - **`pyfinalo_ast`**: AST interpreter (`UntypedAstInterp`) - generates abstract syntax trees
 - **`pyfinalo_explain`**: Explanatory interpreter (`ExplainInterpUntyped`) - provides detailed type checking
 
+### JavaScript/TypeScript Module
+- **`pyfinalo_js`**: Direct value interpreter available as an npm package
```

Comment:
note that the packages is not pushed to https://www.npmjs.com/

Reply:
maybe instructions should be added (not in this section(?)) about how to refer to this npm package (after building)